generators/client/templates/angular/src/main/webapp/app/admin/health/health.component.html.ejs

```diff
@@ -33,15 +33,15 @@
                 </tr>
             </thead>
             <tbody>
-                <tr *ngFor="let health of healthData">
-                    <td><% if (enableTranslation) { %>{{'health.indicator.' + baseName(health.name) | translate}}<% } else { %><span class="text-capitalize">{{ baseName(health.name) }}</span><% } %> {{subSystemName(health.name)}}</td>
+                <tr *ngFor="let healthKey of healthKeys()">
```

Comment:
you can use this I think https://angular.io/api/common/KeyValuePipe

Reply:
Thanks for pointing out!

I tried `keyvalue` pipe, but this changed elements order. Then I used `keys` pipe from `ng-jhipster` (https://github.com/jhipster/ng-jhipster/blob/master/src/pipe/keys.pipe.ts), as this was before my changes, and code is simpler now.